tests/baselines/reference/ambientShorthandFindAllRefs.baseline.jsonc

```diff
@@ -1,167 +1,127 @@
 // === /tests/cases/fourslash/user.ts ===
-// import {/*FIND ALL REFS*/[|x|]} from "jquery";
+// <|import {/*FIND ALL REFS*/[|{| contextId: 0, defId: 0, isWriteAccess: true, isDefinition: true |}x|]} from "jquery";|>
 
+// === Definitions ===
+// === /tests/cases/fourslash/user.ts ===
+// <|import {/*FIND ALL REFS*/[|{| contextId: 0, defId: 0 |}x|]} from "jquery";|>
```

Comment:
It would be nice if the hierarchy between section titles and file names was visually parseable